File: TransCoder_tokenized_test_set_functions/java/FIND_ALL_DIVISORS_OF_A_NATURAL_NUMBER_SET_2.java
static void printDivisors ( int n ) {
  Vector < Integer > v = new Vector < > ( );
  for ( int i = 1;
  i <= Math . sqrt ( n );
  i ++ ) {
    if ( n % i == 0 ) {
      if ( n / i == i ) System . out . printf ( "%d ", i );
      else {
        System . out . printf ( "%d ", i );
        v . add ( n / i );
      }
    }
  }
  for ( int i = v . size ( ) - 1;
  i >= 0;
  i -- ) System . out . printf ( "%d ", v . get ( i ) );
}

File: TransCoder_tokenized_test_set_functions/python/FIND_ALL_DIVISORS_OF_A_NATURAL_NUMBER_SET_2.py
def printDivisors ( n ) :
    list = [ ]
    for i in range ( 1, int ( math . sqrt ( n ) + 1 ) ) :
        if ( n % i == 0 ) :
            if ( n / i == i ) :
                print ( i, end = " " )
            else :
                print ( i, end = " " )
                list . append ( int ( n / i ) )
    for i in list [ : : - 1 ] :
        print ( i, end = " " )
    
